.github/workflows/ci.yml

```diff
@@ -13,9 +13,20 @@ defaults:
     shell: "bash"
 
 env:
+  CI_PRERELEASE: "${{ github.event_name == 'push' }}"
```

Comment:
Would this prerelease for all the commits pushed to a PR? I imagine we'd want to do it only when merging things to `master`

Reply:
Updates to PRs trigger a `pull_request` event. The `push` event is just for pushes to branches properly in the repo—in this workflow, only the master branch triggers `push` events.

Reply:
Great, thanks for the clarification!

This is a little better than I thought, but still if we push other branches (I would push a branch for stuff I'm working on) then we'd get an unwanted release published.

How about matching the name of the branch too?

Reply:
This workflow already only runs when the master branch triggers `push` events; that's line 5.

Reply:
Oh right, it all makes sense now. Thank you!